fdbserver/EncryptKeyProxy.actor.cpp

```diff
@@ -654,6 +680,61 @@ bool isBlobMetadataEligibleForRefresh(const BlobMetadataDetailsRef& blobMetadata
 	return nextRefreshCycleTS > blobMetadata.expireAt || nextRefreshCycleTS > blobMetadata.refreshAt;
 }
 
+ACTOR Future<EKPHealthStatus> getHealthStatusImpl(Reference<EncryptKeyProxyData> ekpProxyData,
+                                                  KmsConnectorInterface kmsConnectorInf) {
+	state UID debugId = deterministicRandom()->randomUniqueID();
+	TraceEvent("EKPHealthCheckStart", ekpProxyData->myId);
```

Comment:
How often will this log line be logged? Do we need this log line Or it could be SevDebug OR even guarded by a #define DEBUG tag?